libs/ui/tooltip/brain/src/lib/brn-tooltip-content.component.ts

```diff
@@ -219,6 +219,11 @@ export class BrnTooltipContentComponent implements OnDestroy {
 		const tooltip = this._tooltip?.nativeElement;
 		if (!tooltip || !this._isBrowser) return;
 		this._renderer2.setStyle(tooltip, 'visibility', isVisible ? 'visible' : 'hidden');
```

Comment:
I think this can be removed then

Reply:
Unfortunately not. I suspect that when a tooltip first appears, the visibility is set to hidden so that the position can be calculated correctly. This happens in line 36. When the tooltip disappears again, it was no longer visible but still present in the same position, which prevented clicking on the elements behind it. That's why I thought that the display should also be set to none. You could also write line 221 like this:

this._renderer2.setStyle(tooltip, 'visibility', isVisible ? 'visible' : '');